packages/auth/src/core/auth/emulator.ts

```diff
@@ -98,7 +103,11 @@ export function connectAuthEmulator(
   authInternal.settings.appVerificationDisabledForTesting = true;
 
   if (!disableWarnings) {
-    emitEmulatorWarning();
+    if (isCloudWorkstation(host)) {
+      updateEmulatorBanner('Auth', true);
```

Comment:
i noticed the others are always tied to the pingServer, where does the auth version of pingServer happen? should we tie to that as well for consistency?